website/content/api-docs/system/internal-ui-namespaces.mdx

```diff
@@ -26,13 +25,40 @@ This endpoint lists the namespaces relevant to the UI.
 
 ```shell-session
 $ curl \
+    --header "X-Vault-Token: ..." \
+    --request GET \
     http://127.0.0.1:8200/v1/sys/internal/ui/namespaces
 ```
 
 ### Sample response
 
 ```json
 {
-  "namespaces": []
+    "request_id": "28a1ca80-cc97-913d-8ed2-844ea501ae0d",
+    "lease_id": "",
+    "renewable": false,
+    "lease_duration": 0,
+    "data": {
+        "keys": [
+            "parent/",
+            "parent/child/",
+            "parent/child/grandchild/",
+            "parent2/"
```

Comment:
Docs folks may have an opinion here but I think we try to have "real world" names for sample code

Reply:
Yeah, we generally try to have something meaningful for example paths. For secret paths, we've used things like `dev/sandbox/` and `dev/production`. For namespaces I don't think we've settled on an explicit pattern yet, but something that reflects common usage (like team hierarchies) would be appropriate. For example `software/` --> `software/eng` --> `software/eng/dev`, `software/eng/qc`